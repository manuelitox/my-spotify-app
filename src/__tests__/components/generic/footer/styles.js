import React from 'react'
import { shallow } from 'enzyme'

import StyledFooter from 'components/generic/Footer/styles'

describe('Component | StyledFooter', () => {
  let component 
  beforeEach( () => {
    component = shallow(
      <StyledFooter />
    )
  })
  it('should render a styled component', () => {
    expect(component).toMatchSnapshot()
  })
})