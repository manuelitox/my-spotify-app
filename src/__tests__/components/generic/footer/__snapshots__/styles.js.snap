// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component | StyledFooter should render a styled component 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  color: ",
          [Function],
          ";
  font-size: 1.2rem;
  font-weight: 500;
  text-align: center;
  aÂ {
    color: ",
          [Function],
          ";
  }
",
        ],
      },
      "displayName": "styled.footer",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "footer",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
